src/QuadForm/Quad/Spaces.jl

```diff
@@ -2357,6 +2366,12 @@ function Base.:(==)(G1::QuadSpaceCls, G2::QuadSpaceCls)
   return all(local_symbol(G1, p) == local_symbol(G2,p) for p in P)
 end
 
+function Base.hash(G::QuadSpaceCls, h::UInt)
+  h = hash(dim(G), h)
```

Comment:
```suggestion
  h = hash(base_ring(G), h)
  h = hash(dim(G), h)
```
the base_ring is used in the first line of `==`, so why not use it here as well